meetings/2018-07-06.md

```diff
@@ -0,0 +1,158 @@
+  The next step is try to figure it out good questions to ask and we would love  your guidance; get the survey out so we can bring
+  it up in the meetings we have with the V8 team.
+* Dan: Ahmad is representing enterprise and has large deployment of node, is our guard Promises a problem, start with this
+  kind of the broadest question.
+* Ahmad: I don’t really hear too much complaints or see kind of situations where like our deployment or our application 
+  development is affected by that, but I think it only becomes a problem in the context of inconsistencies and debugging
+  as we said, it’s not great, but I would echo the point it’s not a burning problem; the nicest thing a developer has to do 
+  in terms of debugging issues when they have to deal with code that’s mixed between Promises and non Promises especially 
+  when it comes down to the native node APIs.
+* Dan: Michael what the current approach TSC is having to dealing with them promised APIs?
+* Michael: At this point its sort of experimental state where we’re at or different people are adding a couple, so far,
+  it’s up to us and DNS it’s sort of experiment and get a feel for what works and what doesn’t as prelude to making more 
+  organized effort to cover the broader surface area. My understanding of the discussion around the debug ability is that 
+  the challenge is, you know, without before Promises if you had an unhandled exception it was easy to track that down because 
+  you could set it up so you got an exception when that happens in the case of Promises because the way I handled rejections
+  respect that’s much more difficult, so if people tasked with tracking down say intermittent or production problems we’re used
+  to that tool which they had that was that they found effective, getting cored out and being able to look at the stock and 
+  then going back the developers, not having that is a challenge for them. I also agree that breaking the spec it’s not 
+  the obvious choice either, so it’s trying to figure it out what will let people debug and identify the location of those
+  problems while maintaining how about we were needed.
+* Benjamin: One of the things that I’ve been seeing it’s that very few people like when I asked the question which is easier
+  for you to debug like Promises the same functions or like Promises without us think about a single weight or callbacks,
+  like almost everyone in the crowd said I can pull the data but almost everyone in the crowd said that a sync functions
+  are easier to debug than callbacks and then Promises were about as hard for the people that I question, this is like 
+  another thing that we can ask, because very few people it’s very predominant in core but I don’t feel that this is the
+  case for user base.
+* Michael: It’s like kind of debugging, we need to make sure we have questions when we go out to capture who we’re talking to,
```

Comment:
Michael: We need to make sure we have questions when we go out to capture who we’re talking to,
+  because I’ve seen quite in a lot of cases it sounded like the developer who writes the code is not necessarily the one
pulled in to debug those issues  in production, so we may talk to a large
+  portion of the users and it’s like well it’s no different because they’re not actually the ones who debug those
+  kind of problems in production. It’s just a matter of understanding, it’s that it’s not an issue but I think we need
+  to get the right people answering as opposed to but it or to be confident that we’re asking the right people before
+  we decide it’s not.